src/core/qgsvectorfilewriter.h

```diff
@@ -287,6 +321,88 @@ class CORE_EXPORT QgsVectorFileWriter
                                             FieldValueConverter* fieldValueConverter = nullptr
                                           );
 
+
+    /** \ingroup core
+     * Options to pass to writeAsVectorFormat()
+     * @note Added in QGIS 3.0
+     */
+    class CORE_EXPORT SaveVectorOptions
```

Comment:
The main driver was that sip errored out with my initial attempt of stuffing 2 more arguments...